src/components/navigation-bar.component.tsx

```diff
@@ -38,13 +43,27 @@ export function NavigationBarComponent() {
 
   const [shown, setShown] = React.useState(false)
   const [language, setLanguage] = React.useState(i18n.language)
+  const [leftHanded, setLeftHanded] = React.useState(false)
```

Comment:
This will get initial state from localStorage
```suggestion
  const [leftHanded, setLeftHanded] = React.useState(() => {
    const leftString = localStorage.getItem('left-handed')
    if (!leftString) return false
    try {
      return JSON.parse(leftString) === true
    } catch {
      return false
    }
  })
```